core/src/test/scala/akka/persistence/r2dbc/query/EventsBySliceSpec.scala

```diff
@@ -47,6 +45,9 @@ object EventsBySliceSpec {
   def config: Config =
     TestConfig.backtrackingDisabledConfig
       .withFallback(ConfigFactory.parseString(s"""
+    # This test is not using backtracking, so increase behind-current-time to
+    # reduce risk of missing events
+    akka.persistence.r2dbc.query.behind-current-time = 500 millis
```

Comment:
this is the attempt to harden it, but I don't have no evidence that it is the reason for the failure